vcluster_versioned_docs/version-0.20.0/configure/vcluster-yaml/control-plane/components/host-path-mapper.mdx

```diff
@@ -15,11 +15,32 @@ Virtual cluster internal logging relies on a separate component called the [Host
 
 If you don't map host paths, logs for vCluster pods are incorrectly resolved by log collectors such as Loki, ELK, and Fluentd.
 
-## vCluster open source
+## vCluster Open Source
 
-Set `enabled: true` to configure vCluster to deploy a HostPath Mapper DaemonSet per Node per vCluster instance.
+### Enabling the HostPath Mapper
+Set `enabled: true` in the vCluster config to allow it to use the HostPath Mapper daemonset, which would be created in the next step.
+```YAML title="vcluster.yaml"
```

Comment:
NIT: usually, the language in code fences is lowercase.

```suggestion
```yaml title="vcluster.yaml"
```

Reply:
updated